src/test/java/org/apache/commons/crypto/stream/AbstractCipherStreamTest.java

```diff
@@ -313,7 +313,6 @@ protected CryptoCipher getCipher(String cipherClass) throws IOException {
 
     @Test
     public void testReadWrite() throws Exception {
-        Assert.assertEquals(null, Openssl.getLoadingFailureReason());
```

Comment:
@winningsix ,  if the underlying engine failed,  OpensslCipher will throw an exception with the loadingFailureReason message. if the native is not loaded, the UT will also failed (an exception is thrown out).
It seems that there is no need to add an interface, which is exposed to client user, to check if the native is loaded successfully. An exception seems enough and friendly for the client user.